src/operations.ts

```diff
@@ -29,7 +29,14 @@ export function parseExpression(exp: string, values: Ref | undefined): any {
 				if (op === 'CURRENCY') {
 					return new Intl.NumberFormat().format(valueA);
 				}
-			} else {
+			} else if (op === "ASUM") {
+                return (
+                    values.value[a] as unknown[]
+                ).reduce((acc, item) => (
```

Comment:
We should check for the existence of `a`. There is a bug when the `displayOnly` option is `true`, if the initial value of the relational field is `null` then it will cause `TypeError: Cannot read properties of undefined (reading 'reduce')`. Because when `displayOnly` is `true`, the `compute` function will be called on the first render.

Reply:
After your fix, I still get the undefined error when `values.value[a]` exists on the object but is undefined. I change it by the `?.` operator and the warnings disappears.